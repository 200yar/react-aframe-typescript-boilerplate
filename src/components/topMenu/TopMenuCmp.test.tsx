import React from 'react';
import ReactDOM from 'react-dom';
import Enzyme, {shallow} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import {shallowToJson} from "enzyme-to-json";
import TopMenu from "./TopMenuCmp";
Enzyme.configure({ adapter: new Adapter() });

describe('TopMenu Component', () => {

  const component = shallow(<TopMenu>Dummy content</TopMenu>);

  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<TopMenu/>, div);
  });

  it('matches snapshot', () => {
    expect(shallowToJson(component)).toMatchSnapshot();
  });

});