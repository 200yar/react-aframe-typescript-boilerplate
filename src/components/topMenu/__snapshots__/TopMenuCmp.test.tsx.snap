// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TopMenu Component matches snapshot 1`] = `
<div
  className="top-menu"
>
  <img
    className="top-menu-icon-left"
    onClick={[Function]}
    src="left-arrow.svg"
  />
  Dummy content
  <img
    className="top-menu-icon-right"
    onClick={[Function]}
    src="burger-icon.svg"
  />
</div>
`;
